src/Raven.Studio/typescript/viewmodels/database/settings/timeSeries.ts

```diff
@@ -141,42 +141,55 @@ class timeSeries extends viewModelBase {
     private initTooltips() {
         popoverUtils.longWithHover($(".aggregation-info"),
             {
-                content: `<ol style="max-width: 400px;">
-                                    <li>
-                                        <small>
-                                            <strong>New time-series data</strong> is created for each aggregation policy that is defined.
-                                            <br />
-                                            Its name will be &lt;original-time-series-name&gt;_&lt;additional-policy-name&gt;
-                                        </small>
-                                    </li>
-                                    <li>
-                                        <small>
-                                            The time-series data generated by the <strong>first</strong> aggregation policy is based on the Raw time-series data.
-                                            After that, the input data for each a aggregation policy will be the time-series data that was generated by the <strong>previous</strong> policy. <br />
-                                            The first Aggregation Rule is based on the Time Series data.<br />
-                                            Each of the following Aggregation Rules is based on data from its <strong>preceding</strong> rule.
-                                        </small>
-                                    </li>
-                                </ol>`,
+                content: `
+                         <small><br>
+                             <ul>
+                                 <li>
+                                     <strong>A new time series</strong> is created by each Aggregation Policy defined.<br>
+                                     Its name will be <code>&lt;Raw-data-time-series-name&gt;@&lt;Aggregation-policy-name&gt;</code>
+                                  </li><br>
+                                  <li>
+                                     Each entry in such auto-generated time series has the following values:<br>
+                                     <i>First, Last, Min, Max, Sum, Count</i><br>
+                                     which is rolled-up data <strong>based only on the Aggregation Time Frame defined by the policy!</strong>
+                                 </li><br>
+                                 <li>
+                                     The data generated by the first aggregation policy is based on the Raw time series data.<br>
+                                     After that, input data for each aggregation policy is the data generated by the previous policy.
+                                 </li><br>
+                                 <li>
+                                     <strong>Important</strong>:<br>
+                                     Aggregated data is not created for time series that have entries with more than 5 values!<br>
```

Comment:
@karmeli87 What if I have 4 values for most of the time, and then by mistake I push TS with 6 values. I assume time series policy won't be applied, right? Will it fail silently? Will it continue to work once I delete 2 values from the entry with 6 values? I assume we should notify user about such case (performance hint in studio?)

Reply:
@karmeli87 @ml054 
Added top warning
![image](https://user-images.githubusercontent.com/17376974/86530584-a0448780-bec2-11ea-9adf-7f4fa1439a24.png)

Reply:
1. The aggregation should stop working.
2. Message in the notification center should appear.
3. After fixing it (delete the extra values) it should resume the work.